packages/ripple-keypairs/src/index.ts

```diff
@@ -88,6 +88,9 @@ function computePublicKeyHash(publicKeyBytes: Uint8Array): Uint8Array {
 }
 
 function deriveAddressFromBytes(publicKeyBytes: Uint8Array): string {
```

Comment:
It occurs to me that these types of checks could be avoided by having an explicit user-defined class for `PublicKey` and `PrivateKey`. The constructor of the class can enforce these checks. But that change requires a lot more effort. 

Thanks for pointing this out!